dapr/conf/helpers.py

```diff
@@ -0,0 +1,54 @@
+from typing import Tuple
+
+
+def parse_endpoint(address: str) -> Tuple[str, str, int]:
```

Comment:
Since this is a pretty important function, how about adding unit tests for this as well?

Add them to `tests/conf/helpers_test.py`

Reply:
EDIT: Let's move the tests you already have in `test_dapr_grpc_client.py` for this here. That should be good enough!